prompting/baseminer/miner.py

```diff
@@ -115,6 +115,7 @@ def __init__(self, config=None, axon=None, wallet=None, subtensor=None):
             forward_fn=self._prompt,
             blacklist_fn=self.blacklist,
             priority_fn=self.priority,
+            required_hash_fields=["messages"],
```

Comment:
Rather than defining the required_hash_fields as `messages`, should this not be `prompting.required_hash_fields`?

Reply:
Rather allowing the servers to define which are the required hash fields, the protocol should dictate the required fields no?

Reply:
Done now inside the protocol and parsed in the axon attach itself instead of passing.